os/board/rtl8730e/src/component/soc/amebad2/misc/ameba_tizenrt_pmu.c

```diff
@@ -240,6 +240,8 @@ void tizenrt_pre_sleep_processing(uint32_t *expected_idle_time)
 	/* ms =x*1000/32768 = (x *1000) >>15 */
 	ms_passed = (u32)((((u64)tick_passed) * 1000) >> 15);
 
+	u16 wakeup_reason = HAL_READ16(SYSTEM_CTRL_BASE_LP, REG_LSYS_DUMMY_090);
```

Comment:
Hi @gSahitya-samsung , this wakeup reason has followed the enum in your PR. But as your PR is not merged, I couldn't feed this directly into the pm_wakehandler. In the meantime, please use it as a patch for your test first.
Thanks.

Reply:
Verified it for LOG_UART interrupts